scripts/backup_outputs.py

```
@@ -0,0 +1,42 @@
+# Get the current working directory
+current_dir = os.getcwd()
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+current_dir = Path(os.getcwd())
+parent_dir = Path(os.path.realpath(__file__)).parent.parent
+
+if current_dir != parent_dir:
+    raise Exception(f"Script can only be run in {parent_dir}")
```

Ruff (0.12.2)

Create your own exception

(TRY002)